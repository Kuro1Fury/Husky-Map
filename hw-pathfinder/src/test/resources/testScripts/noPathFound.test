# Test no path found

CreateGraph A

AddNode A n1
AddNode A n2

FindPath A n1 n2