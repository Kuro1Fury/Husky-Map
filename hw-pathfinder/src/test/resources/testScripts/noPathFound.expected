# Test no path found

created graph A

added node n1 to A
added node n2 to A

path from n1 to n2:
no path found